src/twisted/words/test/test_irc.py

```diff
@@ -1846,55 +1846,82 @@ def test_invalidMaxLength(self):
 
     def test_multipleLine(self):
         """
-        Messages longer than the C{length} parameter to L{IRCClient.msg} will
-        be split and sent in multiple commands.
+        Messages and notices longer than the C{length} parameter to
```

Comment:
Please split these tests apart so that each one tests a single thing: `msg` or `notice`, or convert them to be table driven. The former is probably easier. As they are now constructed they're two tests glommed together, but we want to test the behavior of the API surface separately.